styleguide/source/assets/scss/03-organisms/_ipe-actions.scss

```diff
@@ -0,0 +1,151 @@
+/* Styling for the IPE Actions blocks when viewed in the AMA theme. */
+
+.contextual-region {
+  &:hover div.ipe-actions {
+    background: #f9f9f9;
```

Comment:
can we use variables for colors instead of specific refs here? Perhaps create an ipe-colors.scss with ipe specific colors?

Reply:
I don't think this is worth it unless we end up styling more IPE things in the future.